tests/dragonfly/connection_test.py

```diff
@@ -635,11 +635,11 @@ async def measure(aw):
 
     # Integer set
     await async_client.sadd("set-1", *(i for i in range(100)))
-    assert await measure(async_client.smembers("set-1")) == 1
+    assert await measure(async_client.smembers("set-1")) <= 2
```

Comment:
Please consider adding a brief explanation in the commit message  why the test assertions are now less strict (using <= 2 instead of == 1). This will help reviewers understand the rationale behind allowing more than one reply.